code/backends/CapistranoDeploymentBackend.php

```diff
@@ -154,9 +154,9 @@ public function deploy(
 	/**
 	 * @return array
 	 */
-	public function getDeployOptions() {
+	public function getDeployOptions(DNEnvironment $environment) {
 		return [
-			new PredeployBackupOption(true),
+			new PredeployBackupOption($environment->Usage === 'Production'),
```

Comment:
We should change the magic string to a constant for `Usage`, before we start mispeling `prod`, `Production` or `production`